Gems/Atom/RHI/Code/Include/Atom/RHI/RHISystem.h

```diff
@@ -41,10 +41,14 @@ namespace AZ
             //! Invokes the frame scheduler. The provided callback is invoked prior to compilation of the graph.
             void FrameUpdate(FrameGraphCallback frameGraphCallback);
 
-            // Register/Unregister xr system
+            //! Register/Unregister xr system
             void RegisterXRSystem(XRRenderingInterface* xrRenderingInterface);
             void UnregisterXRSystem();
 
+            //! Get/Set functions for the number of active pipelines in use in a frame 
+            void SetNumActivePipelines(uint16_t numActivePipelines);
```

Comment:
I see what you need. 

I am wondering if it's better to have function call EnablePipelineNameForScope() or something similar

Reply:
In this case Pipeline is a RPI::RenderPipeline. I can update the name to be  SetNumActiveRenderPipelines to avoid confusion. Can you expand on how EnablePipelineNameForScope will be used? Remember that Scopes are created when a render pipeline is loaded and we dont know how many pipelines will be run at that point